added section about roll IPA tokens and page about token issues (#2743)
cloudflare/cloudflare-docs: products/api/src/content/tokens/create/index.md

```

With this you have successfully created an API Token and can now start working with the Cloudflare API!

## Roll API token

> If the secret is lost or believed to be compromised, you can either create a new token or your token can be rolled to regenerate a new secret. Rolling your secret key into a new one will invalidate the previous secret, but the access and permissions will be the same as the previous key.

Reviewer: ```suggestion
If the secret is lost or believed to be compromised, you can either create a new token or your token can be rolled to generate a new secret. Rolling your secret key into a new one will invalidate the previous secret, but the access and permissions will be the same as the previous key.
```
I'd use either `generate a new secret` or `regenerate the secret`.

To roll your API token:

> 1. Log into your [Cloudflare account](https://dash.cloudflare.com) and go to the **API Tokens** section.

Reviewer: ```suggestion
1. Log into your [Cloudflare account](https://dash.cloudflare.com) and go to **User Profile** > **API Tokens**.
```
I couldn't find the section myself, to it's better to add a first location (the user profile), like in the "Create API token" page.

> 1. Click **Roll**.

Reviewer: ```suggestion
1. Next to the API token you wish to roll, click the **three dot icon** > **Roll**.
```
(Tip: I usually try the instructions myself to make sure we're not missing a step in the documentation.)
